src/app_handle.rs

```diff
@@ -577,10 +594,20 @@ impl<V: View> WinHandler for AppHandle<V> {
     }
 
     fn pointer_up(&mut self, event: &glazier::PointerEvent) {
+        self.prev_mouse_pos = MousePosState::None;
         self.event(Event::PointerUp(event.clone()));
     }
 
     fn pointer_move(&mut self, event: &glazier::PointerEvent) {
+        match self.prev_mouse_pos {
+            MousePosState::None => {}
+            MousePosState::Ready => self.prev_mouse_pos = MousePosState::Some(event.pos),
+            MousePosState::Some(prev_pos) => {
+                let position_diff = event.pos - prev_pos;
+                let new_position = self.handle.get_position() + position_diff;
+                self.handle.set_position(new_position);
```

Comment:
I feel like after ```handle_titlebar(true)```, the OS will take care of the window move.

It is only implemented on Windows in Glazier though.

Reply:
It doesn't. I've tested it on windows and it doesn't move. I think it just lets the OS know that it's a titlebar for things like window snapping

Reply:
Although window snapping on windows isn't working either so I'm not sure